plugin/trino-kafka/src/test/java/io/trino/plugin/kafka/TestKafkaIntegrationSmokeTest.java

```diff
@@ -627,4 +627,35 @@ public String toString()
             return tableName; // for test case label in IDE
         }
     }
+
+    @Test
+    public void testApplyFilterAndLimit()
+    {
+        long partitionNum = (long) computeScalar("SELECT count(distinct _partition_id) partition_num FROM kafka.tpch.customer");
```

Comment:
Why `partitionNum` can have different values? How many times this test is executed?